langstream-api-gateway/src/main/resources/application.properties

```diff
@@ -29,4 +29,7 @@ spring.jackson.serialization.order-map-entries-by-keys=true
 application.storage.apps.type=kubernetes
 application.storage.apps.configuration.namespaceprefix=langstream-
 
-application.gateways.code.path=/app/agents
\ No newline at end of file
+application.gateways.code.path=/app/agents
+
+application.topics.producers-cache-enabled=true
+application.topics.producers-cache-size=100
```

Comment:
we should have metrics for this, otherwise it is hard to understand if the system is close to the threshold.
when you hit the threshold we stop caching the system will have very bad performances (latency)